meringue-zest-extension/src/main/java/edu/berkeley/cs/jqf/ZestReplayer.java

```diff
@@ -8,9 +8,11 @@
 import org.junit.runners.model.MultipleFailureException;
 
 import java.io.*;
+import java.lang.reflect.Array;
 import java.util.function.Consumer;
 
 public final class ZestReplayer implements Replayer {
+    private static int maxRecurDepth = 5;
```

Comment:
5 here is an arbitrary value. I can add a java command line parameter for it.

Reply:
Why limit the depth at all?

Reply:
Because if the nested depth is too high (e.g., Array<Array<Array<Array<Array....>>>>), it might cause the calling stack  to overflow. But I don't know if it's a good practice to add this depth limit after all.